### PR TITLE
docs: update broken links and fix small typos

### PR DESCRIPTION
**Pull request by CarinaDraganJW**
## Description

- I updated a few broken links I found while reading through the docs 
- I fixed a few small typos 
- Most of the changes are from formatting 

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: docs/build-from-source.md [hunk 1]
```diff
@@ -81,11 +81,11 @@ These values are then defacto constants, which means code optimizations can remo
...
-You can see an example of how these are used in our [Firebase Live / Preview actions.](https://github.com/jwplayer/ott-web-app/blob/develop/.github/workflows/firebase-live.yml#L14) 
+You can see an example of how these are used in our [Firebase Live / Preview actions.](https://github.com/jwplayer/ott-web-app/blob/develop/.github/workflows/firebase-live.yml#L14)
 
-If building manually, you can create an .env.[mode].local file and add the values there. These files are git ignored which will prevent leaking your secrets to version control. 
+If building manually, you can create an .env.[mode].local file and add the values there. These files are git ignored which will prevent leaking your secrets to version control.
 
-> Note: env variables must begin with the 'APP_' prefix or they are ignored by our vite configuration.
+> Note: env variables must begin with the 'APP\_' prefix or they are ignored by our vite configuration.
```
Comment (AntonLantukh) [inline docs/build-from-source.md:88]: Hm, do we need to change `APP_`?

File: docs/features/shelves-and-libraries.md [hunk 2]
```diff
@@ -82,22 +82,22 @@ Shelves and libraries can be defined in the [app config](/docs/configuration.md)
...
 
-Shelf and library contents are retrieved as a playlist. They can be retrieved with the Playlist API 
+Shelf and library contents are retrieved as a playlist. They can be retrieved with the Playlist API.
 
 ```
 GET Playlist\<playlistid>
 [{
  "title":"Video Title",
- "description":"Lorem ipsum dolor sit amet",  “ 
+ "description":"Lorem ipsum dolor sit amet",  “
```
Comment (AntonLantukh) [inline docs/features/shelves-and-libraries.md]:
```suggestion
 "description":"Lorem ipsum dolor sit amet",
```

File: docs/initialization-file.md [hunk 3]
```diff
@@ -51,6 +51,6 @@ See [.webapp.test.ini](/ini/templates/.webapp.test.ini) for an example.
 
 ### UNSAFE_allowAnyConfigSource
 
-Boolean flag which if true, enables **ANY** 8-character app config ID (or path) to be specified with the [`app-config=<config source>` query param](configuration.md#switching-between-app-configs). 
+Boolean flag which if true, enables **ANY** 8-character app config ID (or path) to be specified with the [`app-config=<config source>` query param](configuration.md#switching-between-app-configs).
 
->***Warning:** Generally the `UNSAFE_allowAnyConfigSource` option should only be used for dev, test, or demo deployments, because it will allow anyone to create URL's that specify any config to be displayed on your domain.* 
+> **\*Warning:** Generally the `UNSAFE_allowAnyConfigSource` option should only be used for dev, test, or demo deployments, because it will allow anyone to create URL's that specify any config to be displayed on your domain.\*
```
Comment (AntonLantukh) [inline docs/initialization-file.md]: Looks like we still get `*` in the formatted string.

File: docs/build-from-source.md (hunk 1 above)
Comment (CarinaDraganJW) [reply docs/build-from-source.md:88]: It's the auto formatting, when I save it modifies it 😨

Comment (CarinaDraganJW) [reply docs/build-from-source.md:88]: The preview seems to ignore the `\` so maybe it's just an auto-formatting behaviour

**Review by AntonLantukh — APPROVED** [review]

**Pull request merged**